lib/ansible/module_utils/compat/importlib.py

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) 2020 Matt Martz <matt@sivel.net>
+# Simplified BSD License (see licenses/simplified_bsd.txt or https://opensource.org/licenses/BSD-2-Clause)
+
+# Make coding more python3-ish
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+import sys
+
+# HACK: keep Python 2.6 controller tests happy in CI until they're properly split
```

Comment:
Now that we're making this a module_util, this should no longer be considered a hack.